samples/bluetooth/peripheral/src/main.c

```diff
@@ -350,6 +350,17 @@ static void hrs_notify(void)
 	bt_hrs_notify(heartrate);
 }
 
+int bt_hrs_ctrl_point_write(uint8_t value)
+{
+	printk("HRS Control point write: %d\n", value);
+
+	return 0;
+}
```

Comment:
I think it would be helpful if this could demonstrate some behavior related to the control point. 

I would also suggest to add an `enum` or `#define`s for the possible values here (or a union of structs if the control points have parameters).

Reply:
added reset_request demo.

I can only find one request in HRS_SPEC_V10.pdf, that is Reset Energy Expanded. and nRF Connect app too only writes 1 to this characteristic. thus used a macro only.